pyproject.toml

```diff
@@ -1,5 +1,123 @@
 [build-system]
-requires = ["setuptools >= 38.0.0",
-            "setuptools_scm >= 2.0.0",
+requires = ["setuptools >= 61.2",
+            "setuptools_scm[toml]>=7.1",
             "wheel"]
-build-backend = 'setuptools.build_meta'
\ No newline at end of file
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = "poppy"
+description = "Physical optics propagation (wavefront diffraction) for optical simulations, particularly of telescopes."
+authors = [{name = "Marshall Perrin", email = "mperrin@stsci.edu"}]
+license = {file = "LISCENCE.md"}
```

Comment:
IMHO it's better to have it as an SPDX ID rather than a reference to the file. BTW license files are captured automatically by `setuptools`.